metadata-ingestion/src/datahub/ingestion/source/s3/config.py

```diff
@@ -70,6 +70,12 @@ class DataLakeSourceConfig(PlatformSourceConfigBase, EnvBasedSourceConfigBase):
         "path_spec", "path_specs", lambda path_spec: [path_spec]
     )
 
+    ingest_patches: bool = Field(
```

Comment:
Suggest having per-aspect knobs for controlling patch-based application or not versus a global one. cc @hsheth2 for your thoughts

Reply:
e.g. 
`patch_properties: true`
`patch_tags: true`

Reply:
Agree that per-aspect knobs is better